doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -1234,7 +1267,17 @@ \subsection{[runtime]}
 needs. For example, the value of an "importance" item could determine how an event 
 handler responds to task failure events.
 
-\paragraph[title]{[runtime] \textrightarrow [[\_\_NAME\_\_]] \textrightarrow [[[meta]]] \textrightarrow title}
+Any suite meta item can now be passed to task event handlers by prefixing the
+string template item name with ``suite\_'', for example : 
+
+\begin{lstlisting}
+[runtime]
+    [[root]]
+        [[[events]]]
+            failed handler = send-help.sh %(suite\_title)s %(suite\_importance)s %(title)s
+\end{lstlisting}
```

Comment:
Underscores shouldn't be escaped with `\` inside the lstlisting environment (the backslashes get printed here).